src/processes/parkingspaces/internal/create-offer.ts

```diff
@@ -76,58 +93,84 @@ export const createOfferForInternalParkingSpace = async (
       return makeProcessError('update-applicant-status', 500)
     }
 
+    const updatedApplicant: DetailedApplicant & { priority: number } = {
+      ...applicant,
+      status: ApplicantStatus.Offered,
+    }
+    const offer = await leasingAdapter.createOffer({
+      applicantId: applicant.id,
+      expiresAt: utils.date.addBusinessDays(new Date(), 2),
+      listingId: listing.id,
+      status: OfferStatus.Active,
+      selectedApplicants: [updatedApplicant, ...restApplicants].map(
+        mapDetailedApplicantsToCreateOfferSelectedApplicants
+      ),
+    })
+
+    if (!offer.ok) {
+      logger.error(
+        offer.err,
+        'Error creating offer for internal parking space - could not create offer'
+      )
+
+      return makeProcessError('create-offer', 500)
+    }
+
+    log.push(`Created offer ${offer.data.id}`)
+    console.log(log)
+    logger.debug(log)
+
     try {
-      const offer = await leasingAdapter.createOffer({
-        applicantId: applicant.id,
-        expiresAt: utils.date.addBusinessDays(new Date(), 2),
-        listingId: listing.id,
-        selectedApplicants: eligibleApplicants,
-        status: OfferStatus.Active,
+      if (!contact.emailAddress)
+        throw new Error('Recipient has no email address')
+
+      await communicationAdapter.sendParkingSpaceOfferEmail({
```

Comment:
Not related to this PR but it would be nice to refactor this in the future to just take the objects as params.